### PR TITLE
RDoc-1126 Client API > operations > server wide > configuration

### PR DESCRIPTION
**Pull request by ml054**

### CODE REVIEW

File: Documentation/4.0/Raven.Documentation.Pages/client-api/operations/server-wide/configuration/get-serverwide-client-configuration.dotnet.markdown [hunk 1]
```diff
@@ -1 +1,21 @@
-﻿TODO
+﻿# Operations : Server : How to Get Server-Wide Client Configuration?
+
+**GetClientConfigurationOperation** is used to return a server-wide client configuration, which is saved on the server and overrides client behavior.
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/client-api/operations/server-wide/configuration/get-serverwide-client-configuration.dotnet.markdown]: GetServerWideClientConfigurationOperation ?

File: Documentation/4.0/Raven.Documentation.Pages/client-api/operations/server-wide/configuration/get-serverwide-client-configuration.dotnet.markdown [hunk 2]
```diff
@@ -1 +1,21 @@
...
+
+{NOTE Client Configuration at database level overrides server-wide client configuration. /}
+
+## Syntax
+
+{CODE config_1_0@ClientApi\Operations\Server\ClientConfig.cs /}
+
+| Return Value | |
+| ------------- | ---- |
+| [ClientConfiguration](../../../glossary/ClientConfiguration) | configuration which will be used by the client API |
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/client-api/operations/server-wide/configuration/get-serverwide-client-configuration.dotnet.markdown]: client API -> RavenDB Client

File: Documentation/4.0/Raven.Documentation.Pages/client-api/operations/server-wide/configuration/get-serverwide-client-configuration.dotnet.markdown [hunk 3]
```diff
@@ -1 +1,21 @@
-﻿TODO
+﻿# Operations : Server : How to Get Server-Wide Client Configuration?
+
+**GetClientConfigurationOperation** is used to return a server-wide client configuration, which is saved on the server and overrides client behavior. 
+
+{NOTE Client Configuration at database level overrides server-wide client configuration. /}
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/client-api/operations/server-wide/configuration/get-serverwide-client-configuration.dotnet.markdown]: Client Configuration at database level -> Client Configuration defined at database level?

File: Documentation/4.0/Raven.Documentation.Pages/client-api/operations/server-wide/configuration/put-serverwide-client-configuration.dotnet.markdown [hunk 4]
```diff
@@ -1 +1,18 @@
-﻿TODO
+﻿# Operations : How to Put Server-Wide Client Configuration?
+
+**PutServerWideClientConfigurationOperation** is used to save server-wide client configuration on the server. It allows you to override client's settings remotely. 
+
+{NOTE Client Configuration at database level overrides server-wide client configuration. /}
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/client-api/operations/server-wide/configuration/put-serverwide-client-configuration.dotnet.markdown]: Client Configuration at database level -> Client Configuration defined at database level?

Comment (arekpalinski) [reply Documentation/4.0/Raven.Documentation.Pages/client-api/operations/server-wide/configuration/put-serverwide-client-configuration.dotnet.markdown]: As Client Configuration is a class in out API we can put it as `ClientConfiguration` ( wrapped by ` chracter)

Comment (ml054) [reply Documentation/4.0/Raven.Documentation.Pages/client-api/operations/server-wide/configuration/put-serverwide-client-configuration.dotnet.markdown]: done

File: Documentation/4.0/Raven.Documentation.Pages/client-api/operations/server-wide/configuration/get-serverwide-client-configuration.dotnet.markdown (hunk 3 above)
Comment (ml054) [reply Documentation/4.0/Raven.Documentation.Pages/client-api/operations/server-wide/configuration/get-serverwide-client-configuration.dotnet.markdown]: done

File: Documentation/4.0/Raven.Documentation.Pages/client-api/operations/server-wide/configuration/get-serverwide-client-configuration.dotnet.markdown (hunk 1 above)
Comment (ml054) [reply Documentation/4.0/Raven.Documentation.Pages/client-api/operations/server-wide/configuration/get-serverwide-client-configuration.dotnet.markdown]: done

**Comment by ml054** [comment]
@arekpalinski Please recheck

**Pull request merged**